pkgs/development/python-modules/opentelemetry-instrumentation-aiohttp-client/default.nix

```
@@ -0,0 +1,61 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, hatchling
+, opentelemetry-api
+, opentelemetry-instrumentation
+, opentelemetry-semantic-conventions
+, opentelemetry-test-utils
+, opentelemetry-util-http
+, wrapt
+, pytestCheckHook
+, aiohttp
+}:
+
+buildPythonPackage rec {
+  pname = "opentelemetry-instrumentation-aiohttp-client";
+  version = "0.39b0";
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "open-telemetry";
+    repo = "opentelemetry-python-contrib";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-HFDebR3d1osFAIlNuIbs5s+uPeTTJ1xkz+BpE5BpciU=";
+    sparseCheckout = [ "/instrumentation/${pname}" ];
+  } + "/instrumentation/${pname}";
```

```suggestion
  };
  
  sourceRoot = "source/instrumentation${pname}";
```
How about using `sourceRoot`?

---

This would make sense, however I would rather not change one dependency and not the others. let's leave it like that for now and change all the dependencies at the same time if that is ok for you.